src/client/components/app/Transactions/components/TxTokenInput.tsx

```diff
@@ -311,6 +311,7 @@ export const TxTokenInput: FC<TxTokenInputProps> = ({
                 readOnly={readOnly}
                 error={inputError}
                 type="number"
+                aria-label={headerText}
```

Comment:
nit: We appear to be using this prop a bit inconsistently, refer to https://github.com/yearn/yearn-finance-v3/pull/610/files#r856877715

Reply:
We will need another PR if we want to include this fixes.
 I will not stop this PR for technical changes that doesn't affect functionality tho.